TYPO3.Flow/Scripts/flow.php

```diff
@@ -20,6 +20,10 @@
 }
 
 if (isset($argv[1]) && ($argv[1] === 'typo3.flow:core:setfilepermissions' || $argv[1] === 'flow:core:setfilepermissions' || $argv[1] === 'core:setfilepermissions')) {
+    $perms = decoct(fileperms(__DIR__. '/setfilepermissions.sh') & 0777);
```

Comment:
we usually try to avoid abbrevations and there needs to be a space between concatenated strings. So this should be sth like
```
$filePermissions = decoct(fileperms(__DIR__ . '/setfilepermissions.sh') & 0777);
```